### DISCUSSION TITLE
Help, the Downloader dosnt find all Content

### DISCUSSION THREAD

**Discussion by xDasDummeKindx** [Avnsx/fansly#196] [Q&A]
Hello, i think i have a problem with the downloader. I am subscribed to a content creater and in the newest post are over 90 pictures and videos but the downloader can only find 5 pictures. How can i fix that? I dont wanna miss the rest of the content (same thing happens to other post as well)

**Comment by xDasDummeKindx** [comment]
here is a screenshot that shows an example Post and how many pictures it finds in it (there are more than 26 and it only finds 5) 
![image](https://github.com/Avnsx/fansly-downloader/assets/57665213/a88d3c52-1e54-4ba1-b284-1ac086744cbc)

**Comment by CanadAce** [comment]
Yes, this just recently happened to me. This program WAS amazing at giving you everything but I suspect some update with Fansly is to blame so that scrapers can only see the preview images, any time it says " +(whatever number)" those images not in the overall summary are the ones not downloaded... I actually came on to see if there was a recent patch to get it back to as good as it was a few weeks ago but it looks like they've all moved on to other things in the last couple of years which is too bad...
